.env.example

```diff
@@ -0,0 +1,13 @@
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_DATABASE=pocketImdb
+DB_USERNAME=root
+DB_PASSWORD=rootroot
+
+JWT_SECRET=SJlde3x4x70Pl00vUjXfGr2zjFB5pJbCdo6fHJ8kTK4PS8RY7WPFjmXLlVNoU9Qmgit
+
+APP_KEY=QeThWmZq4t7w!z%C*F-J@NcRfUjXn2r5
```

Comment:
NIKADA bilo sta sto je secret ne sme da dospe na git

Reply:
sad i da ga obrises, on zauvek ostaje tu, jedino sto mozes da uradis je da promenis taj key sto pre

Reply:
da je to aws key npr do sada bi vec imao da platis par hiljada racun :joy:

Reply:
Slucajno, zurio sam i kopirao ceo .env kao example :D